tests/integration/envrc.rs

```diff
@@ -76,3 +76,21 @@ fn v2_gopath_previously_unset() {
 
     assert_eq!(env.get_env("GOPATH"), DirenvValue::Value("BAR"));
 }
+
+#[test]
+fn bug18_path_with_spaces() {
+    let exemplar = "/mnt/c/Program Files (x86)/QuickTime/QTSystem";
+    let path = env::var("PATH")
+        .map(|x| format!("{}:{}", x, exemplar))
+        .unwrap();
+    let env = EnvrcTestCase::new()
+        .ambient_env("PATH", &path)
```

Comment:
```suggestion
        .ambient_env("PATH", &[exemplar])
```

Would this also work? That way, there is no need to query the actual environment variable PATH with `env::var("PATH")`. If we can avoid pulling parts of the actual environment into our tests, it makes the tests more resilient against changes in the environment (personal machines vs. CI, for example).

Reply:
That's what I meant to suggest in https://github.com/target/lorri/pull/363#discussion_r399084948, but perhaps I didn't make it very clear then.

Reply:
It won't accept a slice, and without appending to the existing PATH I get
```